model/labels/labels.go

```diff
@@ -373,6 +373,29 @@ func (ls Labels) ReleaseStrings(release func(string)) {
 	}
 }
 
+// ExtractNames returns an array of all Name in ls.
+func (ls Labels) ExtractNames() []string {
+	names := make([]string, len(ls))
+	for i, label := range ls {
+		names[i] = label.Name
+	}
+	return names
+}
+
+// Method to check if the Labels contains any of the provided label Names
+// returns true if any of the provided label names are present in the Labels
+// return false if no Names are provided
+func (ls Labels) Contains(names ...string) bool {
```

Comment:
One could argue we don't need this method. 2 reasons:

1. Just because you use it once, does not mean it's useful to have a common helper for everyone. For one use, you can inline this, especially such a simple case.
2. We don't need this code, there is simpler and faster way (suggested in below comments).

Reply:
Good point, I just wansn't sure where to put this at the time.

Will remove after trying out your implementation for `CreatedTimestamp()`

Reply:
removed